poetry.lock

```diff
@@ -932,766 +1681,6 @@ docs = ["Sphinx", "sphinx_rtd_theme"]
 grpc = ["grpcio", "protobuf"]
 
 [metadata]
-lock-version = "1.1"
+lock-version = "2.0"
```

Comment:
I guess this is ok since the repo's GitHub workflows use Poetry 1.4, which supports the new lock format.